ADAL/src/telemetry/ADTelemetry.m

```diff
@@ -134,18 +129,18 @@ - (void)stopEvent:(NSString*)requestId
         [_eventTracking removeObjectForKey:key];
     }
     
-    [_dispatcher receive:requestId event:event];
+    for (ADDefaultDispatcher *dispatcher in _dispatchers)
```

Comment:
do we need to consider protecting `_dispatchers` from race condition?
`_dispatchers` could be being modified by `[registerDispatcher:]` while being read here :)